L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigis_cfi.py

```diff
@@ -73,10 +73,17 @@
                                          runME31Up = True,
                                          runME41Up = True)
 )
-## update shower thresholds for 2025 runs
-from Configuration.Eras.Modifier_run3_CSC_2025_cff import run3_CSC_2025
-run3_CSC_2025.toModify( cscTriggerPrimitiveDigis,
-                      showerParams = showerPSet_2025.clone()
+
+## update shower thresholds for 2025 runs Eras A-E
+from Configuration.Eras.Modifier_run3_CSC_2025_cff import run3_CSC_2025_AtoE
+run3_CSC_2025_AtoE.toModify( cscTriggerPrimitiveDigis,
+                      showerParams = showerPSet_2025_AtoE.clone()
+)
+
+## update shower thresholds for 2025 runs Eras F-G
+from Configuration.Eras.Modifier_run3_CSC_2025_cff import run3_CSC_2025_FtoG
+run3_CSC_2025_FtoG.toModify( cscTriggerPrimitiveDigis,
+                      showerParams = showerPSet_2025_FtoG.clone()
 )
```

Comment:
@AlexAlbert41 Similar to how showerPSet_2025 is imported [here](https://github.com/cms-sw/cmssw/blob/ceebf9ab704dac7c8748071224185159709755f8/L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigis_cfi.py#L14), you need to rename `showerPSet_2025` to `showerPSet_2025_AtoE ` and import the new `showerPSet_2025_FtoG`